### DISCUSSION TITLE
does harfbuzz support OTC files?

### DISCUSSION THREAD

**Discussion by LogicalError** [harfbuzz/harfbuzz#4592] [Q&A] [answered]
I couldn't find any reference to OTC files in the documentation or source code.
If it's not supported, is this planned?

**Comment by behdad** [comment, marked as answer]
HarfBuzz supports OTC files just fine. That's the `face_index` parameter to `hb_face_create()`.

**Comment by LogicalError** [comment]
It might be good to update the documentation then

**Reply by behdad** [reply]
https://harfbuzz.github.io/fonts-and-faces.html#fonts-and-faces-objects
https://harfbuzz.github.io/harfbuzz-hb-face.html#hb-face-create
